src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -1050,7 +1050,7 @@ else if (needsPrepare)
     }
 
     @Override
-    public final java.sql.ResultSetMetaData getMetaData() throws SQLServerException {
+    public final java.sql.ResultSetMetaData getMetaData() throws SQLServerException, SQLTimeoutException {
```

Comment:
Is adding SQLTimeoutException necessary here?

Reply:
Yes.  `buildExecuteMetaData()` previously caught and wrapped the SQLTimeoutException in another exception with less info.  That's been changed so that it's no longer caught, and it gets thrown directly.  The JDBC API allows for throwing this exception type, so it shouldn't be rethrown with less info.